admin/ucf-degree-config.php

```diff
@@ -0,0 +1,285 @@
+<?php
+/**
+ * Registers UCF Degrees options
+ **/
+if ( ! class_exists( 'UCF_Degree_Config' ) ) {
+	class UCF_Degree_Config {
+		public static
+			$option_prefix = 'ucf_degree_',
+			$options_defaults = array(
+				'rest_api' => false
+			);
+
+		/**
+		 * Determines if the WP Rest API is installed and activated
+		 * @author Jim Barnes
+		 * @since 0.0.1
+		 * @return bool
+		 **/
+		public static function rest_api_enabled() {
+			return is_plugin_active( 'rest-api/plugin.php' );
+		}
+
+		/**
+		 * Text to display if rest api is not enabled
+		 * @author Jim Barnes
+		 * @since 0.0.1
+		 * @return void | echos the content
+		 **/
+		public static function rest_api_message() {
+			ob_start();
+			if ( ! self::rest_api_enabled() ) :
+		?>
+			<p style="background-color: #428bca; color: #fff; padding: 24px 48px;">
```

Comment:
Consider adding a class for this, or maybe using Wordpress's built-in alert classes (.error or .notice)